opm/simulators/flow/ReservoirCouplingMaster.cpp

```diff
@@ -66,24 +66,20 @@ void ReservoirCouplingMaster::spawnSlaveProcesses([[maybe_unused]]int argc, char
         slave_argv[1] = nullptr;
         auto num_procs = slave.numprocs();
         std::vector<int> errcodes(num_procs);
-        MPI_Info info;
-        MPI_Info_create(&info);
         const std::string log_file = fmt::format("{}.log", slave_name);
-        // TODO: We need to decide how to handle the output from the slave processes
-        //    For now we just redirect the output to a log file with the name of the slave
-        MPI_Info_set(info, "output", log_file.c_str());
-        MPI_Info_set(info, "error", log_file.c_str());
+        // TODO: We need to decide how to handle the output from the slave processes..
+        //    As far as I can tell, open MPI does not support redirecting the output
+        //    to a file, so we might need to implement a custom solution for this
```

Comment:
@akva2 Any ides on how to handle stdout/stderr from the slaves? Currently it is mixed with output from the master and the other slaves in the terminal which is not ideal..